doc/FAQ.rst

```diff
@@ -126,11 +126,10 @@ another application.
 **Q: Can I do screen scraping with this thing?**
 
 A: That depends. If your application just does line-oriented output then
-this is easy. If it does screen-oriented output then it may work, but it
-could be hard. For example, trying to scrape data from the 'top' command
-would be hard. The top command repaints the text window.
-
-I am working on an ANSI / VT100 terminal emulator that will have methods
-to get characters from an arbitrary X,Y coordinate of the virtual screen.
-It works and you can play with it (see :mod:`pexpect.ANSI`), but I have
-no working examples at this time. 
+this is easy. If a program emits many terminal sequences, from video
+attributes to screen addressing such as those typically using curses(3x), then
```

Comment:
It is the manual section. I do this convention to indicate when the phrase is discussing something with a pairing manual on your system, for example, to describe using 'cat(1)' instead of 'cat'.